src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int BOUND_MAX = 10000;
+    private static final int FIGURES_QUANTITY = 5;
+
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int randomIndex = random.nextInt(FIGURES_QUANTITY);
+        ColorSupplier colorSupplier = new ColorSupplier();

```

Comment:
You should've make this variable a constant - we don't need to initialize the color supplier every time the method is called